src/libLookup/Lookup.cpp

```diff
@@ -1825,6 +1974,21 @@ void Lookup::CommitTxBlocks(const vector<TxBlock>& txBlocks) {
         txBlock.GetHeader().GetBlockNum(), serializedTxBlock);
   }
 
+  bool getStateFromSeedInVacuous = false;
+  if (m_syncType == SyncType::NEW_SYNC ||
+      m_syncType == SyncType::NEW_LOOKUP_SYNC) {  // only for new node joining
+    // Get the state-delta for all txBlocks from random lookup nodes
+    GetStateDeltasFromSeedNodes(lowBlockNum, highBlockNum);
+
+    std::unique_lock<std::mutex> cv_lk(m_mutexSetStateDeltaFromSeed);
+    if (cv_setStateDeltasFromSeed.wait_for(
+            cv_lk, std::chrono::seconds(GETSTATEDELTAS_TIMEOUT_IN_SECONDS)) ==
+        std::cv_status::timeout) {
+      LOG_GENERAL(WARNING, "Didn't receive statedeltas!");
```

Comment:
If the node missed the first time for fetching delta, will the querying in the next try includes the deltas that it failed to fetch previously?

Reply:
Agree. Will try to keep track of missing statedeltas and Try fetching them next time.

Reply:
Fixed. Handled it differently.